<?xml version="1.0"?>

<launch> 
    <node pkg="off_mission" type="off_mission_node" name="off_mission_node" output="screen">
        <rosparam command="load" file="$(find off_mission)/launch/waypoints_xyzy.yaml"/>
        <param name="uav_id" value="1"/>
        <param name="takeoff_height" value="1.2"/>
        <param name="nav_acc_radius" value="0.04"/>
        <param name="nav_acc_angle" value="0.1"/>
    </node>
</launch>
